src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -272,24 +272,27 @@ def remove_filesystem(self, fs_name, confirm):
         command = {'prefix': 'fs rm', 'fs_name': fs_name, 'yes_i_really_mean_it': True}
         return self.mgr.mon_command(command)
 
-    def create_mds(self, fs_name):
-        spec = orchestrator.StatelessServiceSpec(fs_name)
+    def create_mds(self, fs_name, host):
+        spec = orchestrator.StatelessServiceSpec(fs_name,
+            placement=orchestrator.PlacementSpec(nodes=[host]))
```

Comment:
By default, `host` is an empty string, see `_cmd_fs_volume_create`:

```
$  . .tox/py3/bin/activate
$  UNITTEST=true python 
Python 3.6.8 (default, Aug 20 2019, 17:12:48) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tests
>>> import orchestrator
>>> orchestrator.PlacementSpec(nodes=['']).nodes
[HostSpec(hostname='', network='', name='')]
```

Having an empty `HostSpec` doesn't really make sense. Can we make it an empty list, if host is empty?